src/main/java/dev/morling/jfrunit/EnableEvent.java

```diff
@@ -32,18 +33,16 @@
      */
     String value() default "";
 
-//    String name() default "";
-
-    StacktracePolicy stackTrace() default StacktracePolicy.DEFAULT;
+    StacktracePolicy stackTrace() default StacktracePolicy.INCLUDED;
```

Comment:
See my comment below, I believe the `DEFAULT` member is needed to not implictly enable stacktraces for events which have it disabled by default.